### PR TITLE
Translate the question into Chinese

### PR DESCRIPTION
**Pull request by KieSun**
Your repo is awesome!

I want to translate the repo into Chinese.

Now, I finish the 1 - 10 question.

### CODE REVIEW

File: README-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,1288 @@
+# JavaScript 进阶问题列表
+
+我在我的 [Instagram](https://www.instagram.com/theavocoder) 上每天都会发布多选的 JavaScript 问题，并且同时也会在这个仓库中发布。
```
Comment (crimx) [inline README-zh_CN.md]: 中文表述一般是 JavaScript 多选题

File: README-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,1288 @@
...
+
+从基础到进阶，测试你有多了解 JavaScript，刷新你的知识，或者帮助你的 coding 面试！
+:muscle: :rocket：我每周周会在这个仓库下更新新的问题。
```
Comment (crimx) [inline README-zh_CN.md]: 每周周会 typo

**Comment by KieSun** [comment]
finish the 11 - 15 question.

**Comment by KieSun** [comment]
finish the 34 - 43 question.

**Comment by KieSun** [comment]
finish the 16- 20 question.

**Comment by KieSun** [comment]
The translation has been completed. Next, we will update it regularly based on your content.

**Comment by KieSun** [comment]
During the translation, we found a mistake of the No.28 answer.

> `String` is a built-in constructor, which we can add properties to. I just added a method to its prototype. Primitive strings are automatically converted into a string object, generated by the string prototype function. So, all strings (string objects) have access to that method!

The string object is generated by `toObject` function and it's provisional.

**Comment by lydiahallie** [comment]
Please let me know once it's completed and ready to be merged (it's hard for me to proofread, haha)

**Pull request merged**